recipe-server/client/control_new/routes.js

```diff
@@ -22,16 +24,16 @@ const routes = {
         crumb: 'View Recipe',
       },
     },
-    '/extensions': {
+    '/extension': {
```

Comment:
I can't get the nested routing to work with a trailing slash added to `/extension`. I think only leaf nodes can safely have a trailing slash. We might have to just ship without trailing slashes anywhere for the first release.